assets/js/blocks/checkout/inner-blocks/checkout-order-summary-taxes/edit.tsx

```diff
@@ -63,6 +64,7 @@ export const Edit = ( {
 							/>
 						</PanelBody>
 					) }
+				<CheckoutSidebarCompatibilityNotice />
```

Comment:
Same here! We should place the alert under the Name & description panel before the Block's settings panels (i.e., Let's move it to line 39).